src/otx/recipe/detection/atss_mobilenetv2_tile.yaml

```diff
@@ -0,0 +1,189 @@
+# @package _global_
+defaults:
+  - override /trainer: default
+  - override /base: detection
+  - override /callbacks: detection
+  - override /data: mmdet
+  - override /model: mmdet
+data:
+  tile_config:
+    enable_tiler: true
+    grid_size:
+      - 2
+      - 2
+  train_subset:
+    batch_size: 8
+    transforms:
+      - type: LoadImageFromFile
+      - type: LoadAnnotations
+        with_bbox: true
+      - type: MinIoURandomCrop
+        min_ious:
+          - 0.1
+          - 0.3
+          - 0.5
+          - 0.7
+          - 0.9
+        min_crop_size: 0.3
+      - type: Resize
+        scale:
+          - 992
+          - 736
+        keep_ratio: false
+      - type: RandomFlip
+        prob: 0.5
+      - type: PackDetInputs
+        meta_keys:
+          - ori_filename
+          - flip_direction
+          - scale_factor
+          - gt_ann_ids
+          - flip
+          - ignored_labels
+          - ori_shape
+          - filename
+          - img_shape
+          - pad_shape
+  val_subset:
+    batch_size: 8
+    transforms:
+      - type: LoadImageFromFile
+      - type: Resize
+        scale:
+          - 992
+          - 736
+        keep_ratio: false
+      - type: LoadAnnotations
+        with_bbox: true
+      - type: PackDetInputs
+        meta_keys:
+          - ori_filename
+          - scale_factor
+          - ori_shape
+          - filename
+          - img_shape
+          - pad_shape
+  test_subset:
+    batch_size: 8
+    transforms:
+      - type: LoadImageFromFile
+      - type: Resize
+        scale:
+          - 992
+          - 736
+        keep_ratio: false
+      - type: LoadAnnotations
+        with_bbox: true
+      - type: PackDetInputs
+        meta_keys:
+          - ori_filename
+          - scale_factor
+          - ori_shape
+          - filename
+          - img_shape
+          - pad_shape
+model:
+  otx_model:
+    config:
```

Comment:
Most of the `otx_models` now don't have a `config` argument, which seems to be an old-version model config. This seems to be causing integration tests to fail.